apps/backend/src/datasources/postgres/ExperimentDataSource.ts

```diff
@@ -573,25 +572,45 @@ export default class PostgresExperimentDataSource
     offset?: number,
     sortField?: string,
     sortDirection?: string,
-    searchText?: string
```

Comment:
suggestion: I think we should move `searchText` and `instrumentScientistUserId` inside the filter, wdyt?